templates/taxi/driver_list.html

```diff
@@ -7,7 +7,7 @@
       {% for driver in driver_list%}
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          ({{ driver.first_name }} {{ driver.last_name }}){% if driver.id == user.id %}(Me){% endif %}
```

Comment:
Ensure that the `user` variable is available in the template context to avoid errors when comparing `driver.id` with `user.id`.